### ISSUE TITLE
uglifyjs no longer registered

### ISSUE THREAD

**Issue by robertdenton** [Trott/grunt-html-smoosher#22]
Hi there. Getting an error when I try and install your task. According to the [uglifyjs npm page](https://www.npmjs.com/package/uglifyjs) it was unpublished Feb. 9, 2015.

```
npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "grunt-html-smoosher" "--save-dev"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.18
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/uglifyjs
npm ERR! 404 
npm ERR! 404 'uglifyjs' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'grunt-html-smoosher'
```

**Comment by lonnie-smith** [comment]
Had the same problem. Pull request #21 seems to address the issue.

**Comment by Trott** [comment]
I published grunt-html-smoosher-install-fix to npm with the change from the PR #21. Change `grunt-html-smoosher` in your package.json and your Gruntfile.js to `grunt-html-smoosher-install-fix` and you should be good to go. At least, that worked for me. 

I don't plan on keeping it up to date or anything (I didn't even bother updating the README) so hopefully this gets fixed here or else someone else forks it and maintains it. Then I can deprecate that thing.

**Comment by Trott** [comment]
This is now fixed in grunt-html-smoosher 0.1.4.
